analysis/grammar_search_analysis.py

```diff
@@ -190,8 +190,7 @@ def _main() -> None:
 
     outdir = os.path.join(os.path.dirname(os.path.realpath(__file__)),
                           "results")
-    if not os.path.exists(outdir):
-        os.makedirs(outdir)
+    os.makedirs(outdir, exist_ok=True)
```

Comment:
this is a better pattern that we should prefer going forward to avoid coverage issues @tomsilver

Reply:
cool